packages/blob/src/index.ts

```diff
@@ -112,93 +112,24 @@ export async function head(
   return mapBlobResult(headResult);
 }
 
-// vercelBlob.list()
```

Comment:
i moved all the list stuff into a new file. I can undo this, but it felt like this file was growing too large

Reply:
Would you be open to do so in a first PR, then add your PR on top of it?
If you feel like moving all methods I am fine too, it's a good idea.

This will provide a better blame once merged into main.

Reply:
good idea 👍 PR: https://github.com/vercel/storage/pull/478